src/GeoBounds.cpp

```diff
@@ -0,0 +1,21 @@
+//
+// Created by matth on 30/03/2019.
+//
+
+#include "GeoBounds.h"
+namespace NovelRT {
+GeoBounds::GeoBounds(const NovelRT::GeoVector<float>& position, const NovelRT::GeoVector<float>& size) {
+  _corners[0] = GeoVector<float>(position.getX() - (size.getX() / 2), position.getY() - (size.getY() / 2));

```

Comment:
could this not be a for loop?

Reply:
No - the maths is different for both sides of the bounds.